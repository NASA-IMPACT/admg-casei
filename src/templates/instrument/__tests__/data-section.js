import React from "react"
import { create } from "react-test-renderer"

import DataSection from "../data-section"

const id = "unique-id"
const dois = [
  {
    id: "",
    shortname: "",
    longname: "",
    campaigns: [],
    platforms: [],
  },
]

describe("Data Section", () => {
  it("matches snapshot", () => {
    const tree = create(<DataSection id={id} dois={dois} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
