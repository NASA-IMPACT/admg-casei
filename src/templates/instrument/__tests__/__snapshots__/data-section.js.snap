// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data Section matches snapshot 1`] = `
.c0.c0 {
  display: grid;
  grid-template-columns: repeat(12,1fr);
  -webkit-column-gap: 1rem;
  column-gap: 1rem;
  row-gap: 2rem;
  position: relative;
  margin: 0;
  padding: 0;
  margin-bottom: 6rem;
  background-color: none;
}

.c0.c0 > *,
.c0.c0 h3 {
  color: hsla(0,0%,100%,0.9);
}

.c1.c1 {
  position: absolute;
  top: -80px;
}

.c2.c2 {
  grid-column: 1 / span 12;
}

<section
  className="c0"
  data-cy="unique-id-section"
>
  <div
    className="c1"
    id="unique-id"
  />
  <div
    style={
      Object {
        "alignSelf": "end",
        "gridColumn": "1 / span 7",
      }
    }
  >
    <a
      href="#unique-id"
    >
      <h2>
        Data Products
      </h2>
    </a>
  </div>
  <div
    className="c2"
  >
    <div
      style={
        Object {
          "display": "grid",
          "gap": "1rem",
          "gridTemplateColumns": "repeat(auto-fit, minmax(18rem, 1fr))",
        }
      }
    >
      <div
        data-cy="data-product"
        style={
          Object {
            "backgroundColor": "#303641",
            "display": "flex",
            "flexDirection": "column",
            "gap": "0.5rem",
            "padding": "1.5rem",
          }
        }
      >
        <label
          data-cy="doi-label"
        >
          
        </label>
        <a
          data-cy="doi-link"
          href="http://dx.doi.org/"
          rel="noopener noreferrer"
          style={
            Object {
              "color": "#aac9ff",
            }
          }
          target="_blank"
        >
          
        </a>
        <label
          data-cy="no-campaigns-label"
        >
          No Related Campaigns or Platforms
        </label>
      </div>
    </div>
  </div>
</section>
`;
